### PR TITLE
Add a way to skip lowering to nvprims

### PR DESCRIPTION
**Pull request by IvanYashchuk**
This PR adds `skip_ops` argument to `TorchRefsNvfuserCapabilityMode` and `NvfuserPrimsMode` which is an iterable of function names to be skipped in the translation to nvprims process.

### CODE REVIEW

File: test/test_prims.py [hunk 1]
```diff
@@ -164,6 +165,48 @@ def test_nvfuser_impl_is_used(self, device):
...
+        includes_any_prims_sin = any(
+            node.target == torch.ops.prims.sin.default for node in gm.graph.nodes
+        )
+        self.assertTrue(includes_any_prims_sin)
+        include_any_nvprims_sin = any(
+            node.target == torch.ops.nvprims.sin.default for node in gm.graph.nodes
+        )
+        self.assertFalse(include_any_nvprims_sin)
+
+    def test_skip_ops_nvfuser_capability_mode(self, device):
```
Comment (mruberry) [inline test/test_prims.py:189]: Add a comment for this test

File: torch/_prims/context.py [hunk 2]
```diff
@@ -108,6 +108,9 @@ class NvfuserPrimsMode(torch.overrides.TorchFunctionMode):
     nvprim does not exist.
     """
 
+    def __init__(self, *, skip_ops=()):
```
Comment (mruberry) [inline torch/_prims/context.py:111]: Add a comment describing skip_ops and its intended/expected use

**Review by mruberry — APPROVED** [review]
Test failure looks relevant; code looks good to me but I made some inline comments, mostly requesting additional comments

Once @jjsjann123 has had a chance to review and tests are passing just ping me to stamp this

**Review by jjsjann123 — APPROVED** [review]
LGTM.

Regarding the comment here: https://github.com/pytorch/pytorch/pull/85566#pullrequestreview-1119296900
This is the approach for a blank skip node for a given op. I think this is a nice debugging tool, meanwhile, I feel it's too big a hammer for issues like we see in #85566, where we only want to disable the lowering of a certain op.

**Pull request closed without merging**